link/link.go

```diff
@@ -325,11 +325,13 @@ func (l *RawLink) Info() (*Info, error) {
 		extra = &TracingInfo{}
 	case XDPType:
 		extra = &XDPInfo{}
+	case PerfEventType:
+		// not supported
```

Comment:
So there is extra, but we don't support decoding it? If yes please cut a ticket to track this and add any context you have.